src/blob_storage.cc

```diff
@@ -121,8 +121,9 @@ void BlobStorage::ComputeGCScore() {
     gc_score_.push_back({});
     auto& gcs = gc_score_.back();
     gcs.file_number = file.first;
-    if (file.second->file_size() < cf_options_.merge_small_file_threshold) {
-      gcs.score = 1;
+    if (file.second->file_size() <= cf_options_.merge_small_file_threshold ||
+        file.second->gc_mark()) {
+      gcs.score = cf_options_.blob_file_discardable_ratio;
```

Comment:
why not set to 1 as before?

Reply:
To make them low priority. We want to merge blob files that exceed the discardable ratio first.

Reply:
make sense. better to add an inline comment.